decidim-comments/app/frontend/comments/add_comment_form.component.tsx

```diff
@@ -377,6 +381,7 @@ const AddCommentFormWithMutation = graphql<addCommentMutation, AddCommentFormPro
                 comments: [],
                 hasComments: false,
                 acceptsNewComments: false,
+                userCanComment: false,
```

Comment:
Maybe I'd replace `userCanComment` with `userAllowedToComment`, or something similar. Maybe the user cannot comment because comments are not enabled, but "allowed" is tightly bound to permissions (which is what we're checking here)